packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/interfaces/NotificationsReconstructor.java

```diff
@@ -0,0 +1,13 @@
+package expo.modules.notifications.notifications.interfaces;
+
+import android.os.Parcel;
+
+import expo.modules.notifications.notifications.model.NotificationRequest;
+
+public interface NotificationsReconstructor {
+
+  NotificationRequest reconstructNotificationRequest(Parcel parcel);
+
+  String getName();
```

Comment:
Why did you add the `getName()` function?

Reply:
Just for debugging, removed :)